gulpfile.js

```diff
@@ -1,16 +1,18 @@
-const { src, dest, watch, series } = require('gulp');
-const livereload = require('gulp-livereload');
-
-const pkg = require('./package.json');
-const del = require('del');
-
+import commonjs from '@rollup/plugin-commonjs';
+import resolve from '@rollup/plugin-node-resolve';
+import del from 'del';
+import fs from 'fs';
+import gulp from 'gulp';
```

Comment:
The `package.json` file has the `"type": "module"` flag set, so I ran into compile errors when trying to run the `build` and `dev` scripts. I converted the file to ES6 style imports. Alternatively, I could revert these changes and remove `"type": "module"` from the package file.

Reply:
Ah nice. I'm not running into that for some reason but seems like the right change.